### ISSUE TITLE
`ctpu status` crashes

### ISSUE THREAD

**Issue by mlucy** [tensorflow/tpu#155]
I'm using ctpu version 1.3 downloaded from https://dl.google.com/cloud_tpu/ctpu/latest/linux/ctpu .

I get the following stack trace:

```
mlucy@eve:~$ ctpu status
Your cluster is running!
        Compute Engine VM:  RUNNING
        Cloud TPU:          RUNNING
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x671b7e]

goroutine 1 [running]:
github.com/tensorflow/tpu/tools/ctpu/commands.(*statusCmd).Execute(0xc4200915f0, 0x770040, 0xc4200a0000, 0xc42009e780, 0x0, 0x0, 0x0, 0x6dddc0)
        /tmp/ctpu-release/src/github.com/tensorflow/tpu/tools/ctpu/commands/status.go:214 +0x5ce
github.com/google/subcommands.(*Commander).Execute(0xc4200a4000, 0x770040, 0xc4200a0000, 0x0, 0x0, 0x0, 0x5)
        /tmp/ctpu-release/src/github.com/google/subcommands/subcommands.go:141 +0x29f
github.com/google/subcommands.Execute(0x770040, 0xc4200a0000, 0x0, 0x0, 0x0, 0xc4200b0540)
        /tmp/ctpu-release/src/github.com/google/subcommands/subcommands.go:385 +0x5f
main.main()
        /tmp/ctpu-release/src/github.com/tensorflow/tpu/tools/ctpu/main.go:87 +0xd5e
```

**Comment by saeta** [comment]
Hi @mlucy This should be fixed in `ctpu` version 1.4. Please do re-open this issue if this issue still persists with ctpu 1.4. All the best, -Brennan

**Comment by mlucy** [comment]
Awesome, thank you.
